bench/old-vs-new.js

```diff
@@ -9,6 +9,15 @@ var newZ = require('..');
 var Identity = require('../test/Identity');
 var shuffle = require('../test/shuffle');
 
+//  chainRec :: (Number -⁠> c, Number -⁠> c, Number) -⁠> Array c, Number) -⁠> Array b
+function chainRecArrayNumber(next, done, x) {
+  return [done(x), (x <= 1 ? done : next)(x - 1)];
+}
+
+//  duplicate :: a -> Pair a a
```

Comment:
Let's use `Array2` in place of `Pair`.

Reply:
I copied the style from the other signatures in this file. In that case let's change all.

Reply:
:zap: